site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -8,9 +8,11 @@ Do you want your block to interact with the application using it? Read on.
 
 ## Almost-as-quick start
 
-Blocks are web components. We suggest writing them using React, a ready-made framework for writing web components.
+Blocks are discrete components of functionality. You can write them using React, a ready-made library for writing components.
```

Comment:
I think we still haven't cracked the first-sentence's wording. Is a divider block really providing **functionality**, or is it a stylistic element introduced to improve visual comprehension and/or symbolize separation?

```suggestion
Blocks are discrete components that render information or provide functionality. You can create them using any web technology or framework.
```